projects/gapps-html-to-google-doc/src/HtmlToGoogleDoc.js

```
@@ -0,0 +1,99 @@
+        break;
+      case 'ol':
+        const listItems = element.getChildren('li');
+        listItems.forEach(li => {
+          const listItem = parent.appendListItem(li.getText());
+          listItem.setGlyphType(DocumentApp.GlyphType.NUMBER);
+          listItem.setFontFamily('Arial');
+          listItem.setBackgroundColor(null);
+        });
+        break;
+      case 'table':
+        const rows = element.getChildren('tr');
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)